### ISSUE TITLE
Issue in the command

### ISSUE THREAD

**Issue by BRim28** [ilyak93/GAIN-pytorch#1]
Hello @ilyak93 ,

1. I am not much used to PyTorch.
The command that you mentioned in the readme file:
"python --batchsize=20 --total_epochs=50 ...."
I believe the name of the .py file after 'python' is missing.

2. The model.GAIN and model.batch_GAIN_v2 files are missing. I believe we need to rename some files.

3. Can you please add details about downloading and preprocessing the datasets in the readme file.

**Comment by ilyak93** [comment]
@BRim28, Hi.
1. You can ignore the instructions with the command line, because they are for a closed group of developers, which have the data mentioned in the use-case and it isn't available for the public for now.
2. Fixed it.
3. About the VOC dataset 2012, I'll probably will work on it a little bit more later, I need to comment it a little bit more and make it more developer/user friendly, for now you can download the dataset from the link i posted and then in the file I mentioned you only need to change the path to the dataset folder and run the main file of the VOC dataset. If you look at the dataloader of the VOC, the section of the augmentations is in comments (no preproccesing in this section of the code) and you can do any augmentations that you would like in the function used in the main (preprocess_image) (don't forget to do them also on the masks if you interested in EX use also). I've used there pytorch AutoAugment, look for it in pytorch documents.
If you will go through the MedT code, I've upgraded the whole code to be multi-threaded, so you can be inspired how to do it in your own, I'll probably get to it later also (it accelerating the runtime significantly) .
 
In the VOC dataset I've also didn't implement the EX loss part, as in the other dataset code, but you can look at how it is done and get inspiration from it. 

Generally my main goal was to use is on the other dataset, so the main work is done there and later I will adapt the VOC main code to be more similar to the first, so it will be more readable and efficient.
For now, you can figure it out if you run and explore it I guess.
I'll add more about the data preparation section later also
 (but as I mentioned above, you can just look at the preprocess function).

**Comment by BRim28** [comment]
Hi @ilyak93 !

Thank you for the detailed reply. Surely I will take a closer look and get the code working.

**Comment by ilyak93** [comment]
@BRim28, I added detailed run instructions for the VOC dataset. 
Enjoy :)

**Comment by BRim28** [comment]
Hi @ilyak93 !
Thank you for informing me, I will surely take a look.
Meanwhile, the repository is looking good now. Kudos to your efforts!
